otx/algorithms/action/adapters/__init__.py

```diff
@@ -13,3 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions
 # and limitations under the License.
+
```

Comment:
Is this necessary? When OTXActionCls is imported, user should use command "from otx.algorithms.action.adapters.openvino'